AIDriver.lua

```diff
@@ -230,7 +234,7 @@ function AIDriver:stop(msgReference)
 	self.turnIsDriving = false
 	-- don't delete the collision detector in dev mode so we can see collisions logged while manually driving
 	if not CpManager.isDeveloper then
-		self:deleteCollisionDetector()
+		-- self:deleteCollisionDetector()		-- makes no sense to delete it
```

Comment:
If something isn't needed, delete it, not just comment out, there's enough dead code in CP already...
Also, in this case the entire if construct is unnecessary.

Reply:
Will be deleted.